### ISSUE TITLE
Word Embedding

### ISSUE THREAD

**Issue by aggounix** [guillaumegenthial/tf_ner#22]
Is it possible to use an other word embedding than glove (for other languages)?

**Comment by QianhuiWu** [comment]
> Is it possible to use an other word embedding than glove (for other languages)?

I think it's okay. You just need to change the path of the embedding file in build_glove.py.

**Comment by mraduldubey** [comment]
Yes. That'd be fine @aggounix . I have used fasttext instead of glove. And even more, I have used quantized embeddings to reduce memory requirement.

**Comment by ghost** [comment]
Anyone tried the feature extractor form BERT repo? Was thinking about using the last two layers of BERT concat them and use the 1548 long vector insted of the glove ones.
